src/build/streams.ts

```diff
@@ -17,7 +17,9 @@ import * as stream from 'stream';
 
 import File = require('vinyl');
 
-(Symbol as any).asyncIterator = Symbol.asyncIterator || Symbol('asyncIterator');
+if (typeof (Symbol as any).asyncIterator  === 'undefined') {
```

Comment:
Well i also think it is a bit overly paranoid, but found it in a place in the polymer-cli so thought it was the way to go.

will change it.